src/main/scala/cognite/spark/StringDataPointsRelation.scala

```diff
@@ -26,6 +26,8 @@ case class StringDataPointsFilter(
 class StringDataPointsRelationV1(config: RelationConfig)(override val sqlContext: SQLContext)
     extends DataPointsRelationV1[StringDataPointsItem](config)(sqlContext) {
 
+  import PushdownUtilities.filtersToTimestampLimits
```

Comment:
Unless we need implicits or something this should be move to the top alongside other imports.

Reply:
Makes sense!